services/sidecar/requirements/_base.txt

```diff
@@ -4,22 +4,17 @@
 #
 #    pip-compile _base.in
 #
-aiohttp==3.6.2
 amqp==2.4.2               # via kombu
-async-timeout==3.0.1      # via aiohttp
-attrs==19.3.0             # via aiohttp
 billiard==3.6.0.0         # via celery
 celery==4.3.0
 certifi==2019.3.9         # via minio, requests
-chardet==3.0.4            # via aiohttp, requests
+chardet==3.0.4            # via requests
 decorator==4.4.0          # via networkx
 docker-pycreds==0.4.0     # via docker
 docker==3.7.2
-idna-ssl==1.1.0           # via aiohttp
-idna==2.8                 # via idna-ssl, requests, yarl
+idna==2.8                 # via requests
 kombu==4.5.0
 minio==4.0.16
-multidict==4.5.2          # via aiohttp, yarl
 networkx==2.3
 pika==1.0.1
 psycopg2-binary==2.8.3
```

Comment:
could we update to 2.8.4? it's python 3.8 compatible.

Reply:
done